### PR TITLE
add search result special case

### PR DESCRIPTION
**Pull request by charlesangus**
Search results were not deleting - there is a secondary confirmation page which wasn't being handled. This pull request adds a special case for search results which handles the confirmation page.

### CODE REVIEW

File: deleter.go [hunk 1]
```diff
@@ -197,6 +197,7 @@ type activityReader struct {
 func (actRead *activityReader) ReadItems(year int, month int, category string) {
 	requestURL, sectionIDStr := createRequestURL(year, month, actRead.fbl.profileID, category)
 	output := actRead.req.Request(requestURL)
+	fmt.Sprintf("%s", output)
```
Comment (marcelja) [inline deleter.go:200]: Please remove this line

File: deleter.go [hunk 2]
```diff
@@ -212,6 +213,7 @@ func (actRead *activityReader) ReadItems(year int, month int, category string) {
 		requestURL = strings.SplitAfter(output, searchString)[0]
 		requestURL = facebookURL + requestURL[strings.LastIndex(requestURL, `"`)+1:]
 		output = actRead.req.Request(requestURL)
+		time.Sleep(1 * time.Second)
```
Comment (marcelja) [inline deleter.go:216]: Could you please also remove this line? I think we should make it configurable. But not in this PR.

File: deleter.go [hunk 3]
```diff
@@ -452,6 +454,21 @@ func (del *deleter) DeleteCoverOrProfilePhoto(elem *deleteElement) {
...
+        out := del.req.Request(elem.URL)
+
+	from, to := getURLFromToString(out, "/allactivity/delete/")
+        if from == -1 {
+                return
+        }
+	del_url := facebookURL+out[from:to]
+        out = del.req.Request(del_url)
+        elem.success = true
+
```
Comment (marcelja) [inline deleter.go:469]: Could you run `go fmt` to apply formatting?

**Review by marcelja — APPROVED** [review]
Thanks for that. I didn't notice that the search history is not deleting.

File: deleter.go (hunk 1 above)
Comment (charlesangus) [reply deleter.go]: Whoops, debug print committed, how embarrasing heh.

File: deleter.go (hunk 2 above)
Comment (charlesangus) [reply deleter.go]: Yes, quite right. Could have sworn I removed that before committing but apparently not!

File: deleter.go (hunk 3 above)
Comment (charlesangus) [reply deleter.go]: Oh, neat! First time using Go, didn't know about that.

**Pull request merged**